charts/consul/Chart.yaml

```diff
@@ -16,11 +16,11 @@ annotations:
   artifacthub.io/prerelease: true
   artifacthub.io/images: |
     - name: consul
-      image: docker.mirror.hashicorp.services/hashicorppreview/consul-enterprise:1.17-dev
+      image: docker.mirror.hashicorp.services/hashicorppreview/consul:1.17
     - name: consul-k8s-control-plane
-      image: docker.mirror.hashicorp.services/hashicorppreview/consul-k8s-control-plane:1.3.0-dev
+      image: docker.mirror.hashicorp.services/hashicorppreview/consul-k8s-control-plane:1.3.0
     - name: consul-dataplane
-      image: docker.mirror.hashicorp.services/hashicorppreview/consul-dataplane:1.3-dev
+      image: docker.mirror.hashicorp.services/hashicorppreview/consul-dataplane:1.3
     - name: envoy
       image: envoyproxy/envoy:v1.26.2
   artifacthub.io/license: MPL-2.0
```

Comment:
As a complete aside and non-blocking, I wonder if this license should be considering it includes Consul 🤔

Reply:
We discussed this internally and decided to keep it MPL for now.